### DISCUSSION TITLE
Dataloader for macOS Sequoia

### DISCUSSION THREAD

**Discussion by noam-sch** [forcedotcom/dataloader#1350] [General]
Hey there, ever since updating to the newest macOS version - dataloader won't work.
Is there a planned release date for macOS 15 compatibility?

**Comment by ashitsalesforce** [comment]
Hi @noam-sch , do you see any error on the console when trying to run data loader? Which version of data loader are you trying to run and which version of JRE/JDK are you using?

**Comment by ankeetasingh** [comment]
@ashitsalesforce @noam-sch I got the new mac that came with macOS Sequoia. I downloaded the latest version of dataloader, DataLoader 62.0.2. When trying to open install.command from the finder (Ctrl + Open) as instructed in the documentation, I got the following warning

<img width="257" alt="image" src="https://github.com/user-attachments/assets/74205938-7b4a-4045-940b-70e631809a72">

Then I chatted with support and they told me to run the `./install.command` in terminal. And after making sure that it was not installed in the same folder as the zip file extracted folder, the dataloader was installed and was able to open via the shortcut added in the desktop.

<img width="1065" alt="image" src="https://github.com/user-attachments/assets/94ef9435-3aed-475a-a530-18c404268ca0">

Below is the openJDK info from terminal `java --version`
```
openjdk 21.0.5 2024-10-15 LTS
OpenJDK Runtime Environment Zulu21.38+21-CA (build 21.0.5+11-LTS)
OpenJDK 64-Bit Server VM Zulu21.38+21-CA (build 21.0.5+11-LTS, mixed mode, sharing)
```

**Reply by ashitsalesforce** [reply]
Hi @ankeetasingh , thanks for sharing detailed screenshots and posting a workaround that required you to launch Data Loader from Terminal app. The [installation instructions](https://developer.salesforce.com/docs/atlas.en-us.dataLoader.meta/dataLoader/loader_install_mac.htm) documents the steps to work around this issue without requiring you to launch Data Loader from Terminal app.

**Reply by ankeetasingh** [reply]
@ashitsalesforce In the installation instruction #5.b it is mentioned
<img width="857" alt="image" src="https://github.com/user-attachments/assets/d3231f5e-c719-4af6-9dd7-6c1ee7620a1a">

When I followed that step, instead of open option, I was seeing the following warning and it did nothing. Hence I had to use the terminal to install the dataloader
![image](https://github.com/user-attachments/assets/a3ec9836-a2ab-4471-8d45-0f1a6f31e286)

**Comment by ashitsalesforce** [comment]
Hi @ankeetasingh , thanks for trying. Here is the potential solution I found online: https://www.youtube.com/watch?v=biIvAM94b98

Outline of the steps:
As you try to launch `install.command` and see the dialog, go to Apple System Settings, select "Privacy and Security", go to "Security" section, locate the line stating '"install.command" was blocked to protect your Mac.' and click the button "Open Anyway" next to it. Click on "Open Anyway" button in the dialog that asks you to confirm opening "install.command". 

I unfortunately do not have access to MacOS Sequoia to verify these steps. Will very much appreciate you trying out and providing screenshots of your actual experience.

Thanks.
